src/rust/engine/fs/Cargo.toml

```diff
@@ -24,7 +24,7 @@ lazy_static = "1"
 lmdb = { git = "https://github.com/pantsbuild/lmdb-rs.git", rev = "06bdfbfc6348f6804127176e561843f214fc17f8" }
 log = "0.4"
 parking_lot = "0.6"
-protobuf = { version = "2.0.4", features = ["with-bytes"] }
+protobuf = { version = "2.0.6", features = ["with-bytes"] }
```

Comment:
NB: We already were being floated up to 2.0.6 by cargo (see Cargo.lock) - I just edited these to make it more clear how the carte index patch applies.